NumericOutcome.md

```diff
@@ -91,11 +91,11 @@ If `begin_interval_1` is strictly greater than `0`, then the interval between `0
 ## Contract Execution Transaction Calculation
 
 Given the offerrer's [payout function](PayoutCurve.md), a `total_collateral` amount and [rounding intervals](#rounding-intervals), we wish to compute a list of pairs
-of digits (i.e. arrays of integers) and Satoshi values.
-Each of these pairs will then be turned into a CET whose adaptor point is [computed from the list of integers](CETCompression.md#adaptor-points-with-multiple-signatures) and whose
+of digit prefixes (i.e. arrays of integers) and Satoshi values.
+Each of these pairs will then be turned into a CET whose adaptor point is [computed from the digit prefix](CETCompression.md#adaptor-points-with-multiple-signatures) and whose
```

Comment:
I might misunderstand but normally you can have less CET than adaptor signatures can't you?

Reply:
No there is a 1-1 correspondence between adaptor signatures and CETs as you adaptor sign the CETs?

Reply:
As discussed this is probably not so important.